### ISSUE TITLE
Incremental Backup

### ISSUE THREAD

**Issue by Jubilee101** [pgmoneta/pgmoneta#147]
We would like to add the incremental backup option for pgmoneta. Our eventual hope is to save time and space when doing backup by only receiving changed files and replacing the rest with symlinks. See more details at the discussion forum #146.

**Comment by jesperpedersen** [comment]
This is a potential big task.

We know the LSN of the last backup, so we only need to get the changes since then. How, and how to apply is another matter.

Delta backups are important in order to support bigger installations, so this is a 0.8 task for sure.

**Comment by Jubilee101** [comment]
Yes this is a big task. I'll take my time. I'll start with something easy and let's discuss in the forum about design choices and details.

**Comment by Jubilee101** [comment]
As a starter, we'll work on support for the official delta/incr in postgres 17
